collector/settings.mdx

```diff
@@ -174,6 +174,14 @@ helper functions are expected to be defined, and the one we connect to for serve
       <td>[none]</td>
       <td>Alternative to above, using actual contents of the key</td>
     </tr>
+    <tr>
+      <td><code>DB_ALL_NAMES</code></td>
+      <td>false</td>
+      <td>Alternative to setting <code>*</code> at the last entry of db_name (<code>DB_NAME</code>),
+      monitoring all databases on server. Can be only set using an environment variable.<br />
```

Comment:
"Can be only set" => "Can only be set"

Paging @msakrejda to confirm that this rewording would be more correct.